packages/contentful--cra-template-create-contentful-app/template.json

```diff
@@ -2,9 +2,8 @@
   "package": {
     "dependencies": {
       "@contentful/app-sdk": "^4.3.0",
-      "@contentful/forma-36-fcss": "^0.3.2",
-      "@contentful/forma-36-react-components": "^3.90.7",
-      "@contentful/forma-36-tokens": "^0.11.0",
+      "@contentful/f36-components": "latest",
+      "@contentful/f36-tokens": "latest",
```

Comment:
The `package.json` should always have specific version numbers. We don't want to create new apps using `latest`.